com/metacube/EAD/Assignment 3/src/com/servlets/AddEmployeeServlet.java

```diff
@@ -0,0 +1,53 @@
+package com.servlets;
+
+import java.io.IOException;
+import java.io.PrintWriter;
+
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import com.database.DataAccessObject;
+import com.pojo.Employee;
+
+public class AddEmployeeServlet extends HttpServlet {
+	private static final long serialVersionUID = 1L;
+    
+	protected void doGet(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
+		response.setContentType("text/html");
+		
+		PrintWriter out = response.getWriter();
+		
+		String firstName = request.getParameter("firstName");
+		String lastName = request.getParameter("lastName");
+		String email = request.getParameter("email");
+		int age = (Integer.parseInt(request.getParameter("age")));
```

Comment:
Since its a get request so anyone can alter the url. Thats why validate it for blanks.